src/Axe.Windows.Telemetry/ExceptionExcluder.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+
+namespace Axe.Windows.Telemetry
+{
+    /// <summary>
+    /// Code to allow code to run but ensure that and Exceptions will be excluded
+    /// from the telemetry pipeline
+    /// </summary>
+    public static class ExceptionExcluder
+    {
+        /// <summary>
+        /// Execute a block of code within a try/catch block. If it throws, catch the Exception
+        /// and wrap it in a TelemetryExcludedException, which should be excluded from any
```

Comment:
Please add the [exception](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/exception) tag to the doc comment.

Reply:
Added in iteration 6

---
In reply to: [277696286](https://github.com/Microsoft/accessibility-insights-windows/pull/331#discussion_r277696286) [](ancestors = 277696286)